data/workflow.yaml

```diff
@@ -0,0 +1,300 @@
+-
+  name: dod_workflow
```

Comment:
OK, latest commit addresses this (by means of defaults, not changes to the .yaml)